### PR TITLE
drop Katello.with_remote_execution?

### PR DESCRIPTION
**Pull request by evgeni**
katello depends on REX these days, so we can drop that helper

### CODE REVIEW

File: db/seeds.d/75-job_templates.rb [hunk 1]
```diff
@@ -1,20 +1,18 @@
...
-
-        template.organizations << Organization.unscoped.all if template&.organizations&.empty?
-        template.locations << Location.unscoped.all if template&.locations&.empty?
+User.as_anonymous_admin do
+  JobTemplate.without_auditing do
+    template_files = Dir[File.join("#{Katello::Engine.root}/app/views/foreman/job_templates/**/*.erb")]
+    template_files.reject! { |file| file.end_with?('_ansible_default.erb') } unless Katello.with_ansible?
+    template_files.each do |template|
+      sync = !Rails.env.test? && Setting[:remote_execution_sync_templates]
+      # import! was renamed to import_raw! around 1.3.1
```
Comment (ekohl) [inline db/seeds.d/75-job_templates.rb:7]: Is this a good excuse to also drop this? The gem dependency declares it as `>= 7.1.0` so we don't need to care about those ancient versions.

Comment (evgeni) [reply db/seeds.d/75-job_templates.rb:7]: sounds like a plan!

every `if/else` removed is better readable code :)

**Review by chris1984 — APPROVED** [review]
Looks fine to me. I will ack but want a second look. @jeremylenz did you want to have a second look since you have done more with REX lately?

File: engines/bastion_katello/lib/bastion_katello/engine.rb [hunk 2]
```diff
@@ -34,8 +34,8 @@ class Engine < ::Rails::Engine
         :config_generator =>  lambda do
           { 'consumerCertRPM' => consumer_cert_rpm,
             'defaultDownloadPolicy' => !Foreman.in_rake? && db_migrated && Setting['default_download_policy'],
-            'remoteExecutionPresent' => ::Katello.with_remote_execution?,
-            'hostToolingEnabled' => ::Katello.with_remote_execution?
+            'remoteExecutionPresent' => true,
+            'hostToolingEnabled' => true
```
Comment (ekohl) [inline engines/bastion_katello/lib/bastion_katello/engine.rb:38]: You didn't want to pull the trigger and also drop `remoteExecutionPresent` in the frontend? I was interested so wrote a5337ed2f7ec3255c9bf466f67fe1ab7845214ec. Looks like a nice cleanup.

Comment (evgeni) [reply engines/bastion_katello/lib/bastion_katello/engine.rb:38]: Yeah, it felt like a big can of worms. And I was still hoping for that angular removal

**Review by jeremylenz — APPROVED** [review]
Green CI & lgtm. thanks!

File: engines/bastion_katello/lib/bastion_katello/engine.rb (hunk 2 above)
Comment (ekohl) [reply engines/bastion_katello/lib/bastion_katello/engine.rb:38]: You're being unusually positive with your hope.

**Pull request merged**